components/TextFields/styling/README.md

```diff
@@ -0,0 +1,77 @@
+#  Text field styles
+
+MDC's text fields are designed to be styled by controllers implementing the `MDCTextInputController` protocol. The following controllers are included in the text field component and each one is highly customizeable to express your app's branding and styling.
+
+Some of the classes are marked as "Preferred". These are known, thru extensive UX research, to have big performance gains in usability.
```

Comment:
Grammar nit: "through" instead of "thru". Alternatively something like,  "User Experience (UX) research demonstrates that these styles significantly improve usability." (active voice)

Reply:
Look out for my 'tho's too.

Reply:
Just noticed "customizeable" above should drop the "e" ("customizable")

Reply:
Done.